3-Authorization-II/2-call-api-b2c/API/package.json

```diff
@@ -24,9 +27,9 @@
   },
   "homepage": "https://github.com/Azure-Samples/ms-identity-javascript-react-tutorial#readme",
   "devDependencies": {
-    "jest": "^27.0.6",
-    "nodemon": "^2.0.19",
-    "supertest": "^6.1.4"
+    "jest": "^29.2.1",
+    "nodemon": "^2.0.20",
+    "supertest": "^6.3.0"
```

Comment:
"jest": "^28.1.1",
    "nodemon": "^2.0.16",
    "supertest": "^6.2.3"

Reply:
@salman90 the newer versions are [causing tests to fail](https://github.com/Azure-Samples/ms-identity-javascript-react-tutorial/actions/runs/3379638916/jobs/5611452728) (we should look into updating these eventually)